store/postgres/src/connection_pool.rs

```diff
@@ -86,7 +86,12 @@ impl ForeignServer {
         let password = String::from_utf8(
             config
                 .get_password()
-                .ok_or_else(|| anyhow!("could not find user in `{}`", SafeDisplay(postgres_url)))?
+                .ok_or_else(|| {
+                    anyhow!(
+                        "could not find password in `{}`; you must provide one.",
```

Comment:
We could make this even more helpful if the error became `Could not find password in `{}`. Make sure to use a valid connection URI following https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING`

Reply:
I *think* the doc you linked says password is optional in URI connection strings; it's `graph-node` that is not fully spec-compliant. Am I reading this right?

```
postgresql://[userspec@][hostspec][/dbname][?paramspec]

where userspec is:

user[:password]

...
```

Reply:
Ooh .. yes, you are absolutely right!